src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -89,7 +89,7 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
 
   return (
     <Paper elevation={0} component={BackgroundDiv} sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem' }}>
-      <Typography component={'h3'} variant="h2">
+      <Typography component="h2" variant="h3">
```

Comment:
Fixed semantics. Using component h2 for semantics but variant h3 for styling (smaller font)